README.md

```diff
@@ -25,4 +28,37 @@ namespace DemoApp
         }
     }
 }
-```
\ No newline at end of file
+```
+
+Auth and search for keywords in the inbox and sent folders
+```
+using Toolkit;
+
+namespace DemoApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            //Program.exe random@example.org Passw0rd123
+
+            ExchangeConnection connection = new ExchangeConnection(ExchangeVersion.Exchange2013_SP1);
+            connection.SetConnectionCredentials(args[0], args[1].ToSecureString());
+            connection.SetAutodiscoverUrl(args[0]);
+
+            Search searchData = new Search(connection);
+
+            //Add single search terms
+            searchData.AddBodySearchTerm("Administrator");
+            searchData.AddSubjectSearchTerm("Support");
+
+            //Add multi search terms
+            searchData.AddBodySearchTerms(new string[] { "Password", "Login" });
+            searchData.AddSubjectSearchTerms(new string[] { "Onboarding", "Welcome" });
+
+            List<SearchResult> searchResult = searchData.SearchFolder(FolderName.Inbox | FolderName.SentItems);
```

Comment:
Does this actually work?  I've never tried it 😅

Reply:
😆 You don't test either? 🤣 I guess I could test against the RTO-Lab one. Let me confirm

Reply:
Ahh, gotcha. I've seen it used like this before, so I just assumed. I haven't got around to set up an exchange in my lab yet, so feel free to test it. 🏎️